CONTRIBUTING.md

```diff
@@ -4,37 +4,40 @@ The [Materials Consortia](https://github.com/Materials-Consortia) is very open t
 
 This may be anything from simple feedback and raising [new issues](https://github.com/Materials-Consortia/optimade-python-tools/issues/new) to creating [new PRs](https://github.com/Materials-Consortia/optimade-python-tools/compare).
 
-We have below recommendations for setting up an environment in which one may develop the package further.
+Recommendations for setting up a development environment can be found in the [Installation instructions](https://www.optimade.org/optimade-python-tools/install/#full-development-installation).
```

Comment:
You sure we can do relative links here? It's not a big deal, just wondering.

Reply:
What do you mean? I used the absolute link so that this should work whether its being read from the hosted docs or from the contributing file directly

Reply:
Ah right. Yeah, I just meant to ask if a relative link á la `(install/#full-development-installation)` would work? But it's not crucial.

Reply:
I think it would work in the rendered docs, but not in the CONTRIBUTING.md as rendered by GH, so I'd rather leave it like this.

Reply:
(just because contributing is symlinked in the rendered docs)

Reply:
Okay. It's fine by me. Just wanted to make sure we exhausted all opportunities to use relative links instead of absolute links if possible 👍